src/Components/Notifications/Notification.tsx

```diff
@@ -104,36 +105,50 @@ const Notification: React.FC<NotificationProps> = ({ notificationId }) => {
     return <NotificationErrorMessage />
   }
 
-  switch (notification?.notificationType) {
-    case "ARTWORK_ALERT":
-      return <AlertNotification notification={data.me?.notification} />
-    case "ARTWORK_PUBLISHED":
-      return (
-        <ArtworkPublishedNotification notification={data.me?.notification} />
-      )
-    case "ARTICLE_FEATURED_ARTIST":
-      return (
-        <ArticleFeaturedArtistNotification
-          notification={data.me?.notification}
-        />
-      )
-    case "PARTNER_OFFER_CREATED":
-      return (
-        <PartnerOfferCreatedNotification notification={data.me?.notification} />
-      )
-    case "PARTNER_SHOW_OPENED":
-      return (
-        <PartnerShowOpenedNotification notification={data.me?.notification} />
-      )
-    case "VIEWING_ROOM_PUBLISHED":
-      return (
-        <ViewingRoomPublishedNotification
-          notification={data.me?.notification}
-        />
-      )
-    default:
-      return null
+  const renderNotificationObject = data => {
```

Comment:
The linter should be preventing this?

Reply:
https://github.com/artsy/force/blob/af8878df0bc133ef75b15b591c0df2917d27a6b9/.eslintrc.js#L70

https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/no-unstable-nested-components.md